docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2
+
+### Router/gateway versions
+
+| Library | Supported versions |
+|---------|--------------------|
+| Apollo Router | TODO |
+| `@apollo/gateway` | TODO |
+
+### Directive changes
+
+For details on these directives as defined in Federation 2, see [Federation-specific GraphQL directives](./federated-types/federated-directives/).
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `@key`
+
+</td>
+<td>
+
+Added optional `resolvable` argument.
+
+```graphql
+directive @key(fields: FieldSet!, resolvable: Boolean = true) repeatable on OBJECT | INTERFACE
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `@shareable`
+
+</td>
+<td>
+
+Introduced.
+
+```graphql
+directive @shareable on OBJECT | FIELD_DEFINITION
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `@inaccessible`
+
+</td>
+<td>
+
+Introduced.
+
+```graphql
+directive @inaccessible on
+  | FIELD_DEFINITION
+  | OBJECT
+  | INTERFACE
+  | UNION
+  | ARGUMENT_DEFINITION
+  | SCALAR
+  | ENUM
+  | ENUM_VALUE
+  | INPUT_OBJECT
+  | INPUT_FIELD_DEFINITION
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `@override`
+
+</td>
+<td>
+
+Introduced.
+
+```graphql
+directive @override(from: String!) on FIELD_DEFINITION
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `@link`
+
+</td>
+<td>
+
+Introduced.
+
+```graphql
+directive @link(url: String, as: String, for: link__Purpose, import: [link__Import]) repeatable on SCHEMA
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+##### `@external`, `@requires`, `@provides`, `@extends`
+
+</td>
+<td>
+
+No changes.
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+### Subgraph changes
+
+<table>
+  <thead>
+    <tr>
+      <th>Topic</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+Entities
+
+</td>
+<td>
+
+* Entities no longer _originate_ in a subgraph. Instead, any number of subgraphs can define the same entity and _contribute fields_ to it.
+* Multiple subgraphs _can_ contribute the same field to an entity, _if_ that field is marked as `@shareable` in every subgraph that defines it.
+* Subgraphs no longer need to `extend` (or `@extends`) an entity whenever another subgraph already defines that entity.
+* Each subgraph can apply any number of `@key` directives to an entity.
+* Subgraphs _must_ no longer apply the `@external` directive to their `@key` fields.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+Value types
```

Comment:
Is this still a term in Fed 2? I remember there was a whole conversation around this and now I don't remember how it resolved 😅

Reply:
The conversation resolved with it remaining a term! https://www.apollographql.com/docs/federation/federated-types/sharing-types